code/processes/rdb.q

```diff
@@ -100,6 +102,10 @@ endofday:{[date;processdata]
 	a:{(x;raze exec {(enlist x)!enlist((#);enlist y;x)}'[c;a] from meta x where not null a)}each tables`.;
 	/-save and wipe the tables
 	writedown[hdbdir;date];
+        /-creates new changeset if this is a finspace application
+        if[finspace;
+                    .aws.create_changeset[database;([]input_path:enlist getenv[`KDBSCRATCH];database_path:enlist "/";change_type:enlist "PUT")];
```

Comment:
I guess here is where we add the loop to wait for changeset to be completed? do we need to/are we able to capture some id etc. from the `.aws.create_changeset` call to track the status of the changeset?

Reply:
Yeah, this API returns the changeset ID as part of it's output and there is another aws API (I think its .aws.get_changeset) which checks the current status of a changeset so I think we have everything in place except the HDB reload API